spk/salt-master/old/Makefile

```diff
@@ -0,0 +1,38 @@
+SPK_NAME = salt-master
```

Comment:
Please remove the old one or if there is a good reason you can move it to `/diyspk`

Reply:
Fixed